{"message": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 92, in main\n    input_file = generate.generate(sim_spec=specific_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/generate.py\", line 480, in generate\n    sa.generate_region_inst(sim, sim_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/special_archs.py\", line 113, in generate_region_inst\n    logging.info('region')\nNameError: name 'logging' is not defined\n", "asctime": "2017-06-23 10:35:31,069", "created": 1498232131.0692706, "pathname": "/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py", "funcName": "main", "levelname": "ERROR", "lineno": 95, "process": 4392, "processName": "MainProcess", "exc_info": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 92, in main\n    input_file = generate.generate(sim_spec=specific_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/generate.py\", line 480, in generate\n    sa.generate_region_inst(sim, sim_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/special_archs.py\", line 113, in generate_region_inst\n    logging.info('region')\nNameError: name 'logging' is not defined"}
{"message": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 100, in main\n    with open(jsonfile, 'w') as jf:\nFileNotFoundError: [Errno 2] No such file or directory: '/rickshaw/inputs/rickshaw.json'\n", "asctime": "2017-06-23 10:35:31,069", "created": 1498232131.0699553, "pathname": "/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py", "funcName": "main", "levelname": "ERROR", "lineno": 104, "process": 4392, "processName": "MainProcess", "exc_info": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 100, in main\n    with open(jsonfile, 'w') as jf:\nFileNotFoundError: [Errno 2] No such file or directory: '/rickshaw/inputs/rickshaw.json'"}
{"message": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 92, in main\n    input_file = generate.generate(sim_spec=specific_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/generate.py\", line 480, in generate\n    sa.generate_region_inst(sim, sim_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/special_archs.py\", line 113, in generate_region_inst\n    logging.info('region')\nNameError: name 'logging' is not defined\n", "asctime": "2017-06-23 10:39:56,162", "created": 1498232396.1627111, "pathname": "/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py", "funcName": "main", "levelname": "ERROR", "lineno": 95, "process": 14034, "processName": "MainProcess", "exc_info": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 92, in main\n    input_file = generate.generate(sim_spec=specific_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/generate.py\", line 480, in generate\n    sa.generate_region_inst(sim, sim_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/special_archs.py\", line 113, in generate_region_inst\n    logging.info('region')\nNameError: name 'logging' is not defined"}
{"message": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 100, in main\n    with open(jsonfile, 'w') as jf:\nFileNotFoundError: [Errno 2] No such file or directory: '/rickshaw/inputs/rickshaw.json'\n", "asctime": "2017-06-23 10:39:56,163", "created": 1498232396.163578, "pathname": "/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py", "funcName": "main", "levelname": "ERROR", "lineno": 104, "process": 14034, "processName": "MainProcess", "exc_info": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 100, in main\n    with open(jsonfile, 'w') as jf:\nFileNotFoundError: [Errno 2] No such file or directory: '/rickshaw/inputs/rickshaw.json'"}
{"message": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 92, in main\n    input_file = generate.generate(sim_spec=specific_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/generate.py\", line 480, in generate\n    sa.generate_region_inst(sim, sim_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/special_archs.py\", line 113, in generate_region_inst\n    logging.info('region')\nNameError: name 'logging' is not defined\n", "asctime": "2017-06-23 10:46:47,330", "created": 1498232807.3302503, "pathname": "/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py", "funcName": "main", "levelname": "ERROR", "lineno": 95, "process": 28640, "processName": "MainProcess", "exc_info": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 92, in main\n    input_file = generate.generate(sim_spec=specific_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/generate.py\", line 480, in generate\n    sa.generate_region_inst(sim, sim_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/special_archs.py\", line 113, in generate_region_inst\n    logging.info('region')\nNameError: name 'logging' is not defined"}
{"message": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 101, in main\n    json.dump(input_file, jf, indent=4)\nUnboundLocalError: local variable 'input_file' referenced before assignment\n", "asctime": "2017-06-23 10:46:47,331", "created": 1498232807.3311408, "pathname": "/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py", "funcName": "main", "levelname": "ERROR", "lineno": 104, "process": 28640, "processName": "MainProcess", "exc_info": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 101, in main\n    json.dump(input_file, jf, indent=4)\nUnboundLocalError: local variable 'input_file' referenced before assignment"}
{"message": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 93, in main\n    input_file = generate.generate(sim_spec=specific_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/generate.py\", line 480, in generate\n    sa.generate_region_inst(sim, sim_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/special_archs.py\", line 113, in generate_region_inst\n    logging.info('region')\nNameError: name 'logging' is not defined\n", "asctime": "2017-06-23 10:49:10,397", "created": 1498232950.3972514, "pathname": "/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py", "funcName": "main", "levelname": "ERROR", "lineno": 96, "process": 1272, "processName": "MainProcess", "exc_info": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 93, in main\n    input_file = generate.generate(sim_spec=specific_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/generate.py\", line 480, in generate\n    sa.generate_region_inst(sim, sim_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/special_archs.py\", line 113, in generate_region_inst\n    logging.info('region')\nNameError: name 'logging' is not defined"}
{"message": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 102, in main\n    json.dump(input_file, jf, indent=4)\nUnboundLocalError: local variable 'input_file' referenced before assignment\n", "asctime": "2017-06-23 10:49:10,398", "created": 1498232950.3981788, "pathname": "/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py", "funcName": "main", "levelname": "ERROR", "lineno": 105, "process": 1272, "processName": "MainProcess", "exc_info": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 102, in main\n    json.dump(input_file, jf, indent=4)\nUnboundLocalError: local variable 'input_file' referenced before assignment"}
{"message": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 93, in main\n    input_file = generate.generate(sim_spec=specific_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/generate.py\", line 480, in generate\n    sa.generate_region_inst(sim, sim_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/special_archs.py\", line 122, in generate_region_inst\n    sim['region']['config'] = {\"DeployInst\": generate_deploy_inst(sim)}\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/special_archs.py\", line 143, in generate_deploy_inst\n    for date in months.sort:\nTypeError: 'builtin_function_or_method' object is not iterable\n", "asctime": "2017-06-23 11:11:20,124", "created": 1498234280.124847, "pathname": "/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py", "funcName": "main", "levelname": "ERROR", "lineno": 96, "process": 14977, "processName": "MainProcess", "exc_info": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 93, in main\n    input_file = generate.generate(sim_spec=specific_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/generate.py\", line 480, in generate\n    sa.generate_region_inst(sim, sim_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/special_archs.py\", line 122, in generate_region_inst\n    sim['region']['config'] = {\"DeployInst\": generate_deploy_inst(sim)}\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/special_archs.py\", line 143, in generate_deploy_inst\n    for date in months.sort:\nTypeError: 'builtin_function_or_method' object is not iterable"}
{"message": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 102, in main\n    json.dump(input_file, jf, indent=4)\nUnboundLocalError: local variable 'input_file' referenced before assignment\n", "asctime": "2017-06-23 11:11:20,125", "created": 1498234280.1257746, "pathname": "/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py", "funcName": "main", "levelname": "ERROR", "lineno": 105, "process": 14977, "processName": "MainProcess", "exc_info": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 102, in main\n    json.dump(input_file, jf, indent=4)\nUnboundLocalError: local variable 'input_file' referenced before assignment"}
{"message": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 93, in main\n    input_file = generate.generate(sim_spec=specific_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/generate.py\", line 480, in generate\n    sa.generate_region_inst(sim, sim_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/special_archs.py\", line 122, in generate_region_inst\n    sim['region']['config'] = {\"DeployInst\": generate_deploy_inst(sim)}\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/special_archs.py\", line 143, in generate_deploy_inst\n    for date in months.sort():\nTypeError: 'NoneType' object is not iterable\n", "asctime": "2017-06-23 11:11:40,489", "created": 1498234300.4895215, "pathname": "/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py", "funcName": "main", "levelname": "ERROR", "lineno": 96, "process": 15699, "processName": "MainProcess", "exc_info": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 93, in main\n    input_file = generate.generate(sim_spec=specific_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/generate.py\", line 480, in generate\n    sa.generate_region_inst(sim, sim_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/special_archs.py\", line 122, in generate_region_inst\n    sim['region']['config'] = {\"DeployInst\": generate_deploy_inst(sim)}\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/special_archs.py\", line 143, in generate_deploy_inst\n    for date in months.sort():\nTypeError: 'NoneType' object is not iterable"}
{"message": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 102, in main\n    json.dump(input_file, jf, indent=4)\nUnboundLocalError: local variable 'input_file' referenced before assignment\n", "asctime": "2017-06-23 11:11:40,490", "created": 1498234300.490503, "pathname": "/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py", "funcName": "main", "levelname": "ERROR", "lineno": 105, "process": 15699, "processName": "MainProcess", "exc_info": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 102, in main\n    json.dump(input_file, jf, indent=4)\nUnboundLocalError: local variable 'input_file' referenced before assignment"}
{"message": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 93, in main\n    input_file = generate.generate(sim_spec=specific_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/generate.py\", line 469, in generate\n    temp_arch = generate_archetype(sim_spec, arche, in_commod, out_commod, in_recipe, out_recipe)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/generate.py\", line 417, in generate_archetype\n    vals[name] = {\"val\" : [in_recipe[\"name\"]]}\nTypeError: 'NoneType' object is not subscriptable\n", "asctime": "2017-06-23 11:12:01,129", "created": 1498234321.1294982, "pathname": "/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py", "funcName": "main", "levelname": "ERROR", "lineno": 96, "process": 16357, "processName": "MainProcess", "exc_info": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 93, in main\n    input_file = generate.generate(sim_spec=specific_spec)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/generate.py\", line 469, in generate\n    temp_arch = generate_archetype(sim_spec, arche, in_commod, out_commod, in_recipe, out_recipe)\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/generate.py\", line 417, in generate_archetype\n    vals[name] = {\"val\" : [in_recipe[\"name\"]]}\nTypeError: 'NoneType' object is not subscriptable"}
{"message": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 102, in main\n    json.dump(input_file, jf, indent=4)\nUnboundLocalError: local variable 'input_file' referenced before assignment\n", "asctime": "2017-06-23 11:12:01,130", "created": 1498234321.1304097, "pathname": "/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py", "funcName": "main", "levelname": "ERROR", "lineno": 105, "process": 16357, "processName": "MainProcess", "exc_info": "Traceback (most recent call last):\n  File \"/home/robert/miniconda3/lib/python3.5/site-packages/rickshaw-0.0.1-py3.5.egg/rickshaw/main.py\", line 102, in main\n    json.dump(input_file, jf, indent=4)\nUnboundLocalError: local variable 'input_file' referenced before assignment"}
